src/mtype.d

```diff
@@ -9497,27 +9453,69 @@ public:
         return params;
     }
 
+    extern (D):
```

Comment:
`Parameter` class is already `extern (C++)`, so adding label style `extern (D)` in here is not good.